src/github-handler/branch-handler.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Logger, Octokit, RepoDomain} from '../types';
+
+const refPrefix = 'refs/heads/';
+const masterBranch = 'master';
```

Comment:
Let's future-proof this:
```suggestion
const DEFAULT_PRIMARY_BRANCH = 'master';
```